lib/taskana-data/src/test/java/pro/taskana/sampledata/SampleDataProviderTest.java

```diff
@@ -10,19 +10,19 @@
 
   @Test
   void getScriptsNotNull() {
-    assertThat(SampleDataProvider.getDefaultScripts()).isNotNull();
+    assertThat(SampleDataProvider.getSampleDataCreationScripts()).isNotNull();
     assertThat(SampleDataProvider.getScriptsWithEvents()).isNotNull();
   }
 
   @Test
   void getScriptsNotEmpty() {
-    assertThat(SampleDataProvider.getDefaultScripts().count() > 0).isTrue();
+    assertThat(SampleDataProvider.getSampleDataCreationScripts().count() > 0).isTrue();
```

Comment:
Please replace with
assertThat(SampleDataProvider.getSampleDataCreationScripts().count()).isGreaterThan(0);

-- Can you please also do this for the line below? And maybe if it is somewhere else aswell? That'd be great

Reply:
Actually I haven't changed anything here. I just renamed the method. If you would like to change something, please open a ticket for it.

Reply:
Regarding the logging: this is perfect the way it is as long as we do not create any Strings in the method call parameters.